packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -61,6 +67,10 @@ $s-10: 40px;
   }
 }
 
+.ml-half img {
+  margin-left: 2px !important;
+}
+
```

Comment:
I couldn't declare a spacing variable for 2px since it's supposed to be `0.5` per Tailwind docs and `sass` didn't allow naming a variable `$s-0.5`